test/integration/visitor_can_adjust_quantity_of_item_in_cart_test.rb

```diff
@@ -0,0 +1,44 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.